lib/rollbar_api/rql_jobs.rb

```diff
@@ -0,0 +1,64 @@
+require 'json'
+require 'faraday'
+
+module RollbarAPI
+  API_BASE = "https://api.rollbar.com".freeze
+  FAILED_STATES = %w[failed cancelled timed_out].freeze
+  END_STATES = FAILED_STATES + ["success"]
+
+  class RQLJobs
+    def initialize(api_token = ENV['ROLLBAR_READ_ACCESS_TOKEN'], client = nil)
```

Comment:
I was wondering if we could move away from the pattern of specifying env vars in initializers.. the RollbarNotifierService may not be able to be instantiated without params, as it calls this initializer, which then looks for env vars..
Maybe we could always pass in those bits of config, and only fetch them from env vars at the very top level?
(I think I've done the same elsewhere with the n3rgy, but planning to revisit that..)